src/components/Plot/Plot.rtl.test.js

```diff
@@ -31,7 +31,7 @@ describe('class states', () => {
   })
 
   test('renders standard classes', () => {
-    const img = screen.queryByAltText('')
+    const img = screen.queryByAltText('Empty plot')
```

Comment:
i love it when rtl work improves accessibility 🙌

Reply:
Yeah! It's the mark of a good development tool! 😁